server.lua

```diff
@@ -18,16 +67,16 @@ end
 ---@param amount number
 ---@return boolean
 local function removeMoney(source, amount)
-    if not QBCore then return true end
-    local player = QBCore.Functions.GetPlayer(source)
-    local cashBalance = player.Functions.GetMoney('cash')
-    local bankBalance = player.Functions.GetMoney('bank')
+    if not CoreName then return true end
+    local player = Functions[CoreName].GetPlayer(source)
+    local cashBalance = Functions[CoreName].GetCash(player)
+    local bankBalance = Functions[CoreName].GetBank(player)
 
     if cashBalance >= amount then
-        player.Functions.RemoveMoney('cash', amount, "Customs")
+        Functions[CoreName].RemoveMoney(player, 'cash', amount)
         return true
     elseif bankBalance >= amount then
-        player.Functions.RemoveMoney('bank', amount, "Customs")
+        Functions[CoreName].RemoveMoney(player, 'bank', amount)

```

Comment:
If I'm not mistaken, both qb and esx support reason parameter to add/remove money so you can keep it there